ThirdLesson/PostStruct.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  PostStruct.swift
+//  ThirdLesson
+//
+//  Created by Renat Murtazin on 28.09.2021.
+//
+
+import Foundation
+import UIKit
+
+struct PostStruct {
+    
+    var postImage: UIImage?
```

Comment:
```suggestion
    var image: UIImage?
```
Потмоу что будет post.image, лучше чем post.postImage?